src/components/pipeline-list/pipeline-list.test.js

```diff
@@ -31,6 +37,20 @@ describe('PipelineList', () => {
     }
   );
 
+  test.each(pipelineIDs)(
+    'should apply an active class to a pipeline row only if it is active (id: %s, index: %s)',
+    (id, i) => {
+      const wrapper = setup.mount(<PipelineList onToggleOpen={jest.fn()} />);
+      const { active } = wrapper.find('PipelineList').props().pipeline;
+      const isActive = active === id;
+      const hasClass = wrapper
+        .find('MenuOption')
+        .at(i)
+        .hasClass('pipeline-list__option--active');
+      expect(hasClass).toBe(isActive);
```

Comment:
I think this test could be ideally be written in a much more explicit way as it is potentially a little brittle without having constants for each potential case. Consider breaking this into two tests, one where active is explicitly set as true (e.g. by prop in mount) and one where it is false.

Reply:
I did it this way because I figured it'd be less brittle, as it doesn't rely on knowing which item is active. I could split it into two but I'm not sure that it improve anything, but yeah I'll give it a go!

Reply:
@bru5 done